support/main/default/triggers/AccountMaster.trigger

```diff
@@ -8,7 +8,7 @@ trigger AccountMaster on Account(
   before delete
 ) {
   TriggerHandler handler = new TriggerHandler();
-  handler.bind(TriggerHandler.Evt.beforeInsert, new Handler1());
-  handler.bind(TriggerHandler.Evt.beforeInsert, new Handler2());
+  //handler.bind(TriggerHandler.Evt.beforeUpdate, new Handler1());
+  //handler.bind(TriggerHandler.Evt.beforeInsert, new Handler2());
```

Comment:
why aren't we testing this any more?

Reply:
Ah, got it. Will add this back in.

Reply:
Testing handler 1 as direct binding and handler 2 as auto binding